src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2831,7 +2831,7 @@ protected override INamedTypeSymbol CommonCreateTupleTypeSymbol(ImmutableArray<I
                 typesBuilder.Add(elementTypes[i].EnsureCSharpSymbolOrNull<ITypeSymbol, TypeSymbol>($"{nameof(elementTypes)}[{i}]"));
             }
 
-            return TupleTypeSymbol.Create(typesBuilder.ToImmutableAndFree(), elementNames, this);
+            return TupleTypeSymbol.Create(null, typesBuilder.ToImmutableAndFree(), default(ImmutableArray<Location>), elementNames, this);
```

Comment:
I'd rather add a comment, too many names to add. Next PR.

---
In reply to: [61821306](https://github.com/dotnet/roslyn/pull/11006#discussion_r61821306) [](ancestors = 61821306)